src/Arcus.Templates.WebApi/Startup.cs

```diff
@@ -158,6 +161,12 @@ public void ConfigureServices(IServiceCollection services)
             {
                 swaggerGenerationOptions.SwaggerDoc("v1", openApiInformation);
                 swaggerGenerationOptions.IncludeXmlComments(Path.Combine(AppContext.BaseDirectory, "Arcus.Templates.WebApi.Open-Api.xml"));
+
+#if (ExcludeCorrelation == false)
+                swaggerGenerationOptions.OperationFilter<AddHeaderOperationFilter>("RequestId", "The header that will have an unique identifier to distinguish between requests/responses", false);
+                swaggerGenerationOptions.OperationFilter<AddHeaderOperationFilter>("X-Transaction-Id", "The header that will have an identifier that relate requests/responses", false);
```

Comment:
What about this? Feels more clear, no?
```suggestion
                swaggerGenerationOptions.OperationFilter<AddHeaderOperationFilter>("X-Transaction-Id", "Transaction ID is used to correlate multiple operation calls. A new transaction ID will be generated if not specified.", false);
```